Runtime/Behaviours/CinemachineConfiner2D.cs

```diff
@@ -0,0 +1,511 @@
+#if !UNITY_2019_3_OR_NEWER
+#define CINEMACHINE_PHYSICS_2D
+#endif
+
+using System;
+using System.Collections.Generic;
+using Cinemachine.Utility;
+using UnityEngine;
+using Matrix4x4 = UnityEngine.Matrix4x4;
+using Quaternion = UnityEngine.Quaternion;
+using Vector2 = UnityEngine.Vector2;
+using Vector3 = UnityEngine.Vector3;
+
+namespace Cinemachine
+{
+#if CINEMACHINE_PHYSICS_2D
+    /// <summary>
+    /// <para>
```

Comment:
Added these, because it gives nice structure to the text displayed when we hover over CinemachineConfiner2D. The text here is very big, so I thought it's better if it has some structure.

I could not do it with markdown. Maybe my editor does not support markdown.